src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -518,6 +527,11 @@ const formConfig = {
                       type: 'string',
                       'enum': ['yes', 'no']
                     },
+                    'view:privateRecords4142Notice': {
+                      type: 'object',
+                      'ui:collapsed': true,
```

Comment:
Is `hideIf` not an option here?

Reply:
Probably, but I'm not sure you could use `hideIf` in combination with the standard `expandUnder` functionality, which has a lot of conveniences built in. Also, this approach is what we do for the 'which should I choose?' buttons.